packages/design-system/src/foundation/colors/index.ts

```diff
@@ -2,13 +2,110 @@
 
 import { PaletteOptions, PaletteColor } from "@mui/material/styles";
 import * as base from "./base";
+import * as token from "./token";
 
 export type BaseColor = Record<keyof typeof base.blue, PaletteColor>;
 export type GreyColor = Record<keyof typeof base.grey, PaletteColor>;
 
+interface ColorToken {
+  core: {
+    bg_02: React.CSSProperties["color"];
```

Comment:
위에서 @yami03 님이 말씀하신 것처럼 별도의 타입 파일로 관리될 예정이라면...
상위에 `type CSSPropertyColor = React.CSSProperties["color"]`를 선언해준 뒤 사용하는 것도 좋을 것 같네요.

```suggestion
    bg_02: CSSPropertyColor;
```

Reply:
별로 파일로 분리한 뒤 `CSSPropertyColor` 타입을 새로 선언하도록 수정했습니다.. e0289d0